### DISCUSSION TITLE
Help on installing on HA

### DISCUSSION THREAD

**Discussion by fratz6** [edterbak/HeishaMoNR#162] [Q&A]
Hi, 
someone could please help me install this flow?
I tried for hours but I'm not able to follow the guide, here's my trouble. the guide is still working?

1- In this step what I've to do ? 
"**configuration.yaml**
_example code for creating a virtual climate entity for use in NodeRED
Carefull: Do NOT replace existing configuration.yaml with this one!
If you are running the HomeAssistant Node-Red add-on, you have the install details written here. They differ a little bit compared to a bare Linux install of Node-Red. 1: A persistent folder is automatically present for HomeAssistant. 2: The settings.js is located on a different location._

2- Here I installed the missing packages from the user setting, manage palettes
and I can't find the folder node-red in any directory and neither the file nano settings.js
**Node-Red Addon**
_in HomeAssistant, the configuration section, state the required npm modules.

npm_packages:
  - node-red-dashboard
  - node-red-contrib-moment
  - node-red-contrib-noop
  - node-red-contrib-dashboard-bar-chart-data
Then SSH into your homeassistant instance through Putty or the the SSH Add-on. Type:

cd ..
cd config/node-red
nano settings.js_

Thanks

**Comment by edterbak** [comment]
Sorry for the late reply. 
Im trying to find this out now.

**Reply by edterbak** [reply]
I asked on the tweakers forum about this, but nobody replied. I need to figure it out myself as well. It will not be fast :( sorry.

**Comment by sjengfred** [comment]
@fratz6: The HA guide is a bit outdated. You will have to add the npm packages in the configuration tab of the node-red add-on in HA. 
```
npm_packages:
  - node-red-dashboard
  - node-red-contrib-moment
  - node-red-contrib-noop
  - node-red-contrib-dashboard-bar-chart-data
  - node-red-node-smooth
```

The settings file is in addons_config/xxxxx_nodered. You can find 'xxxxx' by using the 'ls' command in the terminal: 
```
cd addons_config
ls
cd xxxxx_nodered
```
nano settings.js

**Comment by edterbak** [comment]
I have updated the HA section to reflect the above details.

Thank you @sjengfred
